node/src/accountant/scanners/mod.rs

```diff
@@ -894,10 +897,75 @@ impl ReceivableScanner {
             earning_wallet,
             receivable_dao,
             banned_dao,
+            persistent_configuration,
             financial_statistics,
         }
     }
 
+    fn update_start_block_while_no_received_transactions_found(
+        &mut self,
+        new_start_block: u64,
+        logger: &Logger,
+    ) {
+        info!(
```

Comment:
I think this `info` should stay at a higher level and only the logic should stay inside this method.